plugins/faucet/params.go

```diff
@@ -0,0 +1,44 @@
+package faucet
+
+import (
+	"time"
+
+	flag "github.com/spf13/pflag"
+
+	"github.com/gohornet/hornet/pkg/node"
+	iotago "github.com/iotaledger/iota.go/v2"
+)
+
+const (
+	// the amount of funds the requester receives if the target address has less funds than the given amount.
+	CfgFaucetAmount = "faucet.amount"
+	// the amount of funds the requester receives if the target address has more funds than the given amount.
+	CfgFaucetSmallAmount = "faucet.smallAmount"
+	// the maximum allowed amount of funds on the target address.
+	CfgFaucetMaxAddressBalance = "faucet.maxAddressBalance"
+	// the maximum output count per faucet message.
+	CfgFaucetMaxOutputCount = "faucet.maxOutputCount"
+	// the faucet transaction indexation payload.
+	CfgFaucetIndexationMessage = "faucet.indexationMessage"
+	// the maximum duration for collecting faucet batches.
+	CfgFaucetBatchTimeout = "faucet.batchTimeout"
+	// the amount of workers used for calculating PoW when issuing faucet messages.
+	CfgFaucetPoWWorkerCount = "faucet.powWorkerCount"
+)
+
+var params = &node.PluginParams{
+	Params: map[string]*flag.FlagSet{
+		"nodeConfig": func() *flag.FlagSet {
+			fs := flag.NewFlagSet("", flag.ContinueOnError)
+			fs.Int64(CfgFaucetAmount, 10000000, "the amount of funds the requester receives if the target address has less funds than the given amount")
```

Comment:
`has less funds than the given amount` -> -`has less funds than the maximum allowed`

Reply:
We have two thresholds, for example the "amount" is 10 Mi.

If someone has less than 10 Mi, he gets 10 Mi on his address.
If he has more than 10 Mi already, he only gets the small amount (1 Mi) until the max is reached (20 Mi).

Reply:
I know, but your sentence in the help does not make sense. 
what is "the given amount" in that sentence?